static/js/containers/pages/register/RegisterConfirmSentPage.js

```diff
@@ -27,7 +27,9 @@ export class RegisterConfirmSentPage extends React.Component<Props> {
         <div className="std-page-body container auth-page">
           <div className="auth-card card-shadow auth-form">
             <div className="auth-header">
-              <h1>Thank you!</h1>
+              <h1 id="thanks-msg" tabIndex="0">
```

Comment:
do you really think that id is needed as we don't have a specific use of it neither is CS nor somewhere else?